addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -420,6 +471,33 @@ options.Class.include({
         });
         await Promise.all(proms).then(() => $allLinks.remove());
     },
+    /**
+     * @private
+     * @param {JQueryElement} $node
+     * @param {String} content text of the editor
+     */
+    _renderAceEditor($node, content) {
+        this._aceEditor = window.ace.edit($node[0]);
+        this._aceEditor.setTheme('ace/theme/monokai');
+        this._aceEditor.setValue(content, 1)
+        this._aceEditor.setOptions({
+            minLines: 20,
+            maxLines: Infinity,
+            enableBasicAutocompletion: true,
+        });
+        this._aceEditor.renderer.setOptions({
+            highlightGutterLine: true,
+            showInvisibles: true,
+            fontSize: 14,
+        });
+
+        const aceSession = this._aceEditor.getSession();
+        aceSession.setOptions({
+            mode: "ace/mode/xml",
+            useWorker: false,
+        });
+        $node.find('.ace_gutter').attr('aria-hidden', false);
```

Comment:
Note: in the HTML/CSS editor in the customize menu of website, do we do it like that too? Or is the bug there too?

Reply:
It has a custom style here: https://github.com/odoo/odoo/commit/f391b8c5a55bdc512d70cb8b3e61f23903abb5ed#diff-d11b664dcd120bcd3b312ed790c30d86R335-R336

I'll change it through css too then

Reply:
No new scss, the rule should be shared somehow